org.mwc.debrief.lite/.classpath

```diff
@@ -28,5 +28,6 @@
 	<classpathentry combineaccessrules="false" kind="src" path="/org.mwc.cmap.NaturalEarth"/>
 	<classpathentry combineaccessrules="false" kind="src" path="/org.mwc.cmap.openmap"/>
 	<classpathentry combineaccessrules="false" kind="src" path="/org.mwc.debrief.legacy"/>
+	<classpathentry combineaccessrules="false" kind="src" path="/org.mwc.cmap.gt2Plot"/>

```

Comment:
@unnivm - we have to remove this dependency - we can't introduce the dependency on eclipse libs in gt2plot.

Reply:
@unnivm - could you also explain why we depend on NaturalEarth and OpenMap?

thx.

Reply:
@IanMayo ,

Looks these two dependencies(NaturalEarth and OpenMap) are existing. But they are not needed. I have removed from the classpath.

But gt2plot is needed as we are using GtProjection (until now we are using it). This entry is added automatically when we add the Gt2plot project dependency.

What I will remove entries from classpath for gt2plot and make one more commit. But at the time of importing the project, we need to resolve this dependency manually.

Please let me know.